src/hack/update_alert_rules.py

```diff
@@ -0,0 +1,58 @@
+#
```

Comment:
I think we are missing a docs page in this repo describing why we need these scripts, when should we call them and how.